ncm-network/src/main/perl/network.pm

```diff
@@ -265,7 +265,7 @@ sub Configure
     opendir(DIR, $dir_pref);
     # here's the reason why it only verifies eth, bond, bridge, usb and vlan
     # devices. add regexp at will
-    my $dev_regexp='-((eth|seth|em|bond|br|ovirtmgmt|vlan|usb|ib|p\d+p|en(o|(p\d+)?s))\d+|enx[[:xdigit:]]{12})(\.\d+)?';
+    my $dev_regexp='-((eth|seth|em|bond|br|ovirtmgmt|vlan|usb|ib|p\d+p|en(o|(p\d+)?s(?:\d+f)?(?:\d+d)?))\d+|enx[[:xdigit:]]{12})(\.\d+)?';
```

Comment:
@ned21 i'd rather not touch this any more than required. please make remarks in the rewrite, eg https://github.com/quattor/configuration-modules-core/pull/1070/commits/d31aca263e18a64a9544348d21abf9a3f7f07f93